SwedbankPaySDK/Classes/SwedbankPaySDK+Extensions/MerchantBackendConfiguration.swift

```diff
@@ -0,0 +1,207 @@
+            let urlTypeDicts = urlTypes?.lazy.compactMap { $0 as? [AnyHashable : Any] }
+            guard let callbackUrlType = urlTypeDicts?.filter({
+                $0[callbackURLTypeKey] as? Bool == true
+            }).first else {
+                fatalError("Unable to infer callback scheme: No URL type marked as Swedbank Pay callback. Please add a URL type with a unique name, a single unique scheme, and an additional property with the key \(callbackURLTypeKey), type Boolean, and value YES")
+            }
+            guard let schemes = callbackUrlType["CFBundleURLSchemes"] as? [Any],
+                schemes.count == 1,
+                let scheme = schemes[0] as? String
+                else {
+                    fatalError("Unable to infer callback scheme: URL type marked as Swedbank Pay SDK callback does not have exactly one scheme: \(callbackUrlType)")
+            }
+            return scheme
+        }
+                
+        private func withTopLevelResources<T>(
+            _ onFailure: @escaping (Result<T, Error>) -> Void,
+            f: @escaping (TopLevelResources) -> Void
+        ) {
+            rootLink.get(api: api) {
+                switch $0 {
+                case .success(let topLevelResources):
+                    f(topLevelResources)
+                case .failure(let error):
+                    onFailure(.failure(error))
+                }
+            }
+        }
+        
+        public func postConsumers(consumer: SwedbankPaySDK.Consumer?, userData: Any?, completion: @escaping (Result<SwedbankPaySDK.ViewConsumerIdentificationInfo, Error>) -> Void) {
+            guard let consumer = consumer else {
+                fatalError("MerchantBackendConfiguration requires use of Consumer for checkin")
+            }
+            withTopLevelResources(completion) { topLevelResources in
+                topLevelResources.consumers.post(
+                    api: api,
+                    consumer: consumer,
+                    userData: userData
+                ) {
+                    do {
+                        let viewConsumerIdentification = try $0.get().requireOperation(
+                            rel: Operation.TypeString.viewConsumerIdentification.rawValue
+                        )
+                        let info = ViewConsumerIdentificationInfo(
+                            webViewBaseURL: backendUrl,
+                            viewConsumerIdentification: viewConsumerIdentification
+                        )
+                        completion(.success(info))
+                    } catch let error {
+                        completion(.failure(error))
+                    }
+                }
+            }
+        }
+        
+        public func postPaymentorders(paymentOrder: SwedbankPaySDK.PaymentOrder?, userData: Any?, consumerProfileRef: String?, completion: @escaping (Result<SwedbankPaySDK.ViewPaymentOrderInfo, Error>) -> Void) {
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 212 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)